src/drivers/uavcan/rgbled.cpp

```diff
@@ -113,9 +131,74 @@ void UavcanRGBController::periodic_update(const uavcan::TimerEvent &)
 			break;
 		}
 
-		uavcan::equipment::indication::LightsCommand cmds;
 		cmds.commands.push_back(cmd);
 
+	}
+
+	if (_armed_sub.updated()) {
```

Comment:
FYI this publishes at 2 Hz or immediately on any change.

Reply:
That's perfect. I want it to publish even "stale" data semi-frequently in case a light controller resets, etc. Could limit it to 1Hz but the RGB side is already publishing much faster so it doesn't seem worth the fuss.